lib/room.rb

```diff
@@ -1,12 +1,26 @@
-class Rom
-
-  def initialize(category, length, width)
+class Room
+  attr_reader :category
+  def initialize(category, length = 10, width)
```

Comment:
I don't think this makes sense as a default value here-- there's nothing logical to suggest that most rooms have a length of 10.

Reply:
I set it as a default because the starting code had it hard-coded in the initialize method so I thought that was intentional.  This preserved the default setting but also allowed the room to be a different size and pass other tests.  But totally agree... no reason a room should have a length of 10.


class Rom

  def initialize(category, length, width)
    @category = category
    @length = 10
    @width = width
  end

Reply:
i see where you're coming from.  it was intentional in the sense that we wanted you to see that it was causing problems and that it needed to be set equal to the length in the parameter, but there was no particular meaning for the 10 specifically.